ClientCore/SavedGameManager.cs

```diff
@@ -42,7 +40,7 @@ public static List<string> GetSaveGameTimestamps()
 
             for (int i = 0; i < saveGameCount; i++)
             {
-                FileInfo sgFile = SafePath.GetFile(saveGameDirectory, string.Format("SVGM_{0}.NET", i.ToString("D3")));
+                FileInfo sgFile = SafePath.GetFile(saveGameDirectory, $"SVGM_{i:D3}.NET");
```

Comment:
Honestly we are tired of such a change. If you have modified surrounding codes, you can change it but otherwise, I suggest not doing so to reduce possible pains in git rebase/cherrypicks etc. But I guess we can torelate this one for consistency.